src/docfx/template/TemplateEngine.cs

```diff
@@ -210,6 +217,27 @@ private string CreateContent(FilePath file, string? mime, JObject pageModel)
         return ProcessHtml(_errors, file, content);
     }
 
+    private void ProcessConceptualHtml(string html, ConceptualModel conceptual)
+    {
+        var wordCount = 0L;
+
+        var result = HtmlUtility.TransformHtml(html, (ref HtmlReader reader, ref HtmlWriter writer, ref HtmlToken token) =>
+        {
+            HtmlUtility.AddLocaleIfMissingForAbsolutePath(ref token, _locale);
+
+            if (token.Type == HtmlTokenType.Text)
+            {
+                if (!_config.DryRun)
+                {
+                    wordCount += WordCount.CountWord(token.RawText.Span);
```

Comment:
The `WordCount` comment says the logic is shared with loc build. Which does it mean that loc build already has the logic? If so I'd like to keep both the word count patch and the update link locale patch in loc build.

Reply:
If with Wei's PR here, should we consider removing the patch in loc build? That'll make the logic consolidated and avoid future errors.

Reply:
@herohua Yes, we should remove the wordCount logic in loc-build after this PR.

@yufeih Btw, where is the comment?

Reply:
Keep `wordCount` in loading markdown, and recalculate it in loc build.